Engines/Wine/Plugins/Font smoothing/script.js

```diff
@@ -0,0 +1,39 @@
+include("engines.wine.engine.object");
+
+/**
+ * force the Font smoothing
+ * @param {string} mode ("RGB", "BGR" or "Gray Scale")
+ * @returns {Wine} Wine object
+ */
+Wine.prototype.fontSmoothing = function (mode) {
+    var fontSmoothingType;
```

Comment:
Please use `let` instead of `var`:
- `let` indicates a later changeable variable
- `const` indicates a constant value